src/h5web/dimension-mapper/SlicingSlider.tsx

```diff
@@ -25,29 +33,34 @@ function SlicingSlider(props: Props) {
     >
       <span className={styles.label}>D{dimension}</span>
       <ReactSlider
-        // Force refresh when slider height changes - i.e. when Y-axis mapper appears/disappears
-        // https://github.com/zillow/react-slider/issues/172
-        key={`${mapperState.includes('y')}`}
+        ref={sliderRef}
         className={styles.slider}
         ariaLabel="Dimension slider"
-        trackClassName={styles.track}
-        thumbClassName={styles.thumb}
-        renderThumb={(thumbProps, state) => (
-          <div {...thumbProps}>{state.valueNow}</div>
-        )}
-        value={slicingIndex}
-        onChange={(value) => {
-          const newMapperState = [...mapperState];
-          newMapperState[dimension] = value as number;
-          onChange(newMapperState);
-        }}
         min={0}
         max={rawDims[dimension] - 1}
+        step={1}
         marks={height / rawDims[dimension] >= MIN_HEIGHT_PER_MARK}
         markClassName={styles.mark}
-        step={1}
         orientation="vertical"
         invert
+        value={slicingIndex}
+        onChange={(value) => {
+          const newMapperState = [...mapperState];
+          newMapperState[dimension] = value as number;
+          onChange(newMapperState);
+        }}
+        renderThumb={(thumbProps, state) => (
+          <div {...thumbProps} className={styles.thumb}>
+            {state.valueNow}
+          </div>
+        )}
+        renderTrack={({ key }, { index }) =>
+          index === 0 ? (
+            <div key={key} className={styles.track} />
+          ) : (
+            <Fragment key={key} />
+          )
+        }
```

Comment:
Custom render for the track so we only render one track and style it so it doesn't reach the ends of the slider.

Reply:
Can you now put `null` instead of `Fragment` ?

Reply:
Hehe would have been awesome. I've just checked and ReactSlider expect the function to return `Element`.